src/custom/hooks/useUSDCPrice/index.ts

```diff
@@ -194,60 +205,69 @@ export function useCoingeckoUsdPrice(currency?: Currency) {
 
   const tokenAddress = currencyRef.current ? currencyId(currencyRef.current) : undefined
 
+  const chainIdSupported = supportedChainId(chainId)
+  // SWR cache cg usd requests
+  const { data: priceResponse, error: errorResponse } = useSWR<PriceInformation | null>(
+    ['coingeckoUsdPrice', chainIdSupported, tokenAddress],
```

Comment:
nit: why not calling the cache exactly the same name as the function `getUSDPriceQuote`?

Reply:
done